src/Discord.Net.Core/CDN.cs

```diff
@@ -4,8 +4,15 @@ public static class CDN
     {
         public static string GetApplicationIconUrl(ulong appId, string iconId)
             => iconId != null ? $"{DiscordConfig.CDNUrl}app-icons/{appId}/{iconId}.jpg" : null;
-        public static string GetUserAvatarUrl(ulong userId, string avatarId, ushort size, AvatarFormat format)
-            => avatarId != null ? $"{DiscordConfig.CDNUrl}avatars/{userId}/{avatarId}.{format.ToString().ToLower()}?size={size}" : null;
+        public static string GetUserAvatarUrl(ulong userId, string avatarId, ushort size, AvatarFormat format) {
+            if (avatarId == null)
```

Comment:
A user can have a null avatarId if they still have their default avatar. We might be able to handle this better by calculating the default avatar instead of just returning null.